assets/src/components/detours/detourRouteSelectionPanel.tsx

```diff
@@ -52,8 +53,11 @@ export const DetourRouteSelectionPanel = ({
   onSelectRoute,
   onSelectRoutePattern,
   isLoadingRoutePatterns,
+  isRouteInvalid = false,
```

Comment:
note: this seemed like the easiest route, but we _could_ pass in the `snapshot` and `send` values instead if we _wanted_.